tests/container-kill_test.go

```diff
@@ -151,12 +151,25 @@ var _ = Describe("BDD of pod-delete experiment", func() {
 
 			//Checking the chaosresult
 			By("Checking the chaosresult")
-			app, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+experimentName, metav1.GetOptions{})
-			Expect(string(app.Status.ExperimentStatus.Verdict)).To(Equal("Pass"), "Verdict is not pass chaosresult")
+			chaosResultVerdict, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+experimentName, metav1.GetOptions{})
+			fmt.Println("Chaos Result Verdict is: ", chaosResultVerdict.Status.ExperimentStatus.Verdict)
 			Expect(err).To(BeNil(), "Fail to get chaosresult")
+			Expect(string(chaosResultVerdict.Status.ExperimentStatus.Verdict)).To(Equal("Pass"), "Chaos Result Verdict is not Pass")
 
 		})
 	})
+	// BDD for checking chaosengine Verdict
+	Context("Check for chaos engine verdict", func() {
+
+		It("Should check for the verdict of experiment", func() {
+
+			By("Checking the Verdict of Chaos Experiment")
+			chaosEngineVerdict, err := clientSet.ChaosEngines(chaosTypes.ChaosNamespace).Get(engineName, metav1.GetOptions{})
+			Expect(err).To(BeNil(), "Fail to get the chaosengine")
+			fmt.Println("Chaos Engine Verdict is: ", chaosEngineVerdict.Status.Experiments[0].Verdict)
```

Comment:
Instead of 0 can you put a loop and find the experiment by name and check the verdict of that experiment only. Currently, we are using a single experiment in engine but in future, we may use more than one experiment so it would be helpful if you search the experiment instead of hardcoded the index 0

Reply:
not using multiple experiments in engine right now it would be fine to add this when we will add multiple experiment run checks

Reply:
This might be a new test case in the component test in the future?

Reply:
I also said the same thing that in the future we may need this and It is a standard way too. I think it will be straight forward to implement this. Not new test case required for that

Reply:
I know it is straight forward and no new test cases are to be added for this.
I'm telling we will never be having multiple experiment runs in this, this is individual experiment checks. For running multiple experiments in the engine there will be new test cases and in that, we can use such logic to get a verdict.

Reply:
Okay, Tell me only one extra benefit of using a hardcoded value `0` over the derive the verdict by matching the experiment name using loop?
- We are recommended to not use a hardcoded value, we only use if we don't have any option but here we have an easy alternative(std way).

Reply:
It is not only about matching the experiment name. Firstly, the experiment name is a job name we have to get the job of the experiment twice in a BDD only to get the Verdict (once for the verdict check other for updating table) and that also will come in a list so again one task to take out the name from the list so we can't just compare experiment names.
These will be nothing but extra tasks. If there is only one value then not hardcoding it will also not make much difference is what I'm saying.

Reply:
Asked this question to myself during the review. Both points were considered: Need to not hardcode, possibilities of more exp than one in engine, the need to map a result name to a derived exp name. 


In case of multiple exp, not just result but the entire structuring of the bdd should change - right from pulling of all exp at once (not just that inside that one bdd _test.go), loop for logging exp runs, for result and corresponding verification in engine status. 

These are needed but part of more whole refactor. And considering we caught the bug of engine status showing Await when result is Pass pretty later (I have seen some race condition for this in 1.2.2 itself), wanted us to start verifying this asap.

Reply:
cc: @chandankumar4 @ispeakc0de

Reply:
This is okay for now as we having one exp in one result